sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -142,22 +143,31 @@ public async Task StartRecognizeContentPopulatesFormPagePdf(bool useStream)
                 Assert.GreaterOrEqual(cell.ColumnIndex, 0, $"Cell with text {cell.Text} should have column index greater than or equal to zero.");
                 Assert.Less(cell.ColumnIndex, table.ColumnCount, $"Cell with text {cell.Text} should have column index less than {table.ColumnCount}.");
 
-                // There's a single cell in the table (row = 1, column = 3) that has a column span of 2.
-
-                var expectedColumnSpan = (cell.RowIndex == 1 && cell.ColumnIndex == 3) ? 2 : 1;
-
-                Assert.AreEqual(1, cell.RowSpan, $"Cell with text {cell.Text} should have a row span of 1.");
+                // Column = 3 has a column span of 2.
+                var expectedColumnSpan = cell.ColumnIndex == 3 ? 2 : 1;
                 Assert.AreEqual(expectedColumnSpan, cell.ColumnSpan, $"Cell with text {cell.Text} should have a column span of {expectedColumnSpan}.");
 
-                Assert.AreEqual(expectedText[cell.RowIndex, cell.ColumnIndex], cell.Text);
+                // Row = 1 and columns 0-4 have a row span of 2.
+                var expectedRowSpan = (cell.RowIndex == 1 && cell.ColumnIndex != 5) ? 2 : 1;
+                Assert.AreEqual(expectedRowSpan, cell.RowSpan, $"Cell with text {cell.Text} should have a row span of {expectedRowSpan}.");
 
                 Assert.IsFalse(cell.IsFooter, $"Cell with text {cell.Text} should not have been classified as footer.");
                 Assert.IsFalse(cell.IsHeader, $"Cell with text {cell.Text} should not have been classified as header.");
 
                 Assert.GreaterOrEqual(cell.Confidence, 0, $"Cell with text {cell.Text} should have confidence greater or equal to zero.");
-                Assert.LessOrEqual(cell.RowIndex, 1, $"Cell with text {cell.Text} should have a row index less than or equal to one.");
+                Assert.LessOrEqual(cell.RowIndex, 2, $"Cell with text {cell.Text} should have a row index less than or equal to one.");
 
-                Assert.Greater(cell.FieldElements.Count, 0, $"Cell with text {cell.Text} should have at least one field element.");
+                // row = 2, column = 5 has empty text and no elements
+                if (cell.RowIndex == 2 && cell.ColumnIndex == 5)
+                {
+                    Assert.IsEmpty(cell.Text);
+                    Assert.AreEqual(0, cell.FieldElements.Count);
+                }
+                else
+                {
+                    Assert.AreEqual(expectedText[cell.RowIndex, cell.ColumnIndex], cell.Text);
```

Comment:
I'm a bit surprised this works. `expectedText` has dimensions [2, 6], but we get back cells with rowIndex = 2. Shouldn't we get an `ArgumentOutOfRangeException` here?

Reply:
hmm I see the confusion. So the service only returns one cell for `rowIndex=2` which is `(2,5)`. do you think I should be more explicit?

Reply:
I see. It's just testing, so I'm not that concerned.